cmd/go-quai/main.go

```diff
@@ -171,7 +171,7 @@ func init() {
 	// Initialize the CLI app and start Quai
 	app.Action = quai
 	app.HideVersion = true // we have a command to print the version
-	app.Copyright = "Copyright 2013-2021 The go-ethereum Authors"
+	app.Copyright = "Copyright 2013-2023 The go-ethereum and go-quai Authors"
```

Comment:
Not sure we can claim copyright on this back to 2013

Reply:
Removed